app/uk/gov/hmrc/cdsreimbursementclaim/utils/TimeUtils.scala

```diff
@@ -20,6 +20,8 @@ import java.time._
 import java.time.format.DateTimeFormatter
 import scala.util.Try
 
+final case class EisTime(value: String) extends AnyVal
```

Comment:
I can't see anywhere where you have used this class to instantiate a value.

Reply:
FileUploadRequest.scala